external-videos.php

```diff
@@ -47,17 +47,17 @@ public function __construct() {
 
     require_once( ABSPATH . 'wp-admin/includes/taxonomy.php' );
 
-    require_once( plugin_dir_path( __FILE__ ) . 'core/ev-admin.php' );
-    require_once( plugin_dir_path( __FILE__ ) . 'core/ev-helpers.php' );
-    require_once( plugin_dir_path( __FILE__ ) . 'core/ev-widget.php' );
-    require_once( plugin_dir_path( __FILE__ ) . 'core/ev-media-gallery.php' );
-    require_once( plugin_dir_path( __FILE__ ) . 'core/simple_html_dom.php' );
-
-    require_once( plugin_dir_path( __FILE__ ) . 'hosts/ev-youtube.php' );
-    require_once( plugin_dir_path( __FILE__ ) . 'hosts/ev-vimeo.php' );
-    require_once( plugin_dir_path( __FILE__ ) . 'hosts/ev-dotsub.php' );
-    require_once( plugin_dir_path( __FILE__ ) . 'hosts/ev-wistia.php' );
-    require_once( plugin_dir_path( __FILE__ ) . 'hosts/ev-dailymotion.php' );
+    require( plugin_dir_path( __FILE__ ) . 'core/ev-admin.php' );
```

Comment:
Researching the plugin i came across arguments that require_once required significantly more work from the server, and if the code has no chance of double includes, it's always better to require. Now that i'm looking it up again, seems it's not SUCH a big difference. http://stackoverflow.com/questions/186338/why-is-require-once-so-bad-to-use